scene/3d/spring_bone_simulator_3d.cpp

```diff
@@ -1670,7 +1766,9 @@ void SpringBoneSimulator3D::_process_joints(double p_delta, Skeleton3D *p_skelet
 
 		// Convert position to rotation.
 		Vector3 from = current_rot.xform(verlet->forward_vector);
-		Vector3 to = p_inverted_center_transform.basis.xform(next_tail - current_origin).normalized();
+		Vector3 to = p_inverted_center_transform.basis.xform(next_tail - current_origin);
+		from = from.normalized();
+		to = to.normalized();
```

Comment:
If these are on their own line, godot style tends to use a mutator like `from.normalize();` - Just a stylistic choice: it is not important and I am okay either way.